File: frontend/accomodation-booking/src/app/model/address.ts
export class Address {
  country: string = '';
  city: string = '';
  street: string = '';
  number: string = '';

  constructor(country: string, city: string, street: string, number: string) {
    this.country = country;
    this.city = city;
    this.street = street;
    this.number = number;
  }
}

File: accommodation_service/domain/model/address.go
package model

import "go.mongodb.org/mongo-driver/bson/primitive"

type Address struct {
	ID      primitive.ObjectID `bson:"_id,omitempty"`
	Country string             `bson:"country"`
	City    string             `bson:"city"`
	Street  string             `bson:"street"`
	Number  string             `bson:"number"`
}
